### PR TITLE
meta(github): add issue forms

### LINKED ISSUE #8: Setup issue forms

### PR DESCRIPTION
**Pull request by wizard-28**
Closes #8

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      description: Provide your OS version.
+      placeholder: Ubuntu 20.04
+    validations:
+      required: true
+  - type: input
+    id: application
+    attributes:
+      label: Application
+      description: Provide the name of the application you've used LibPacstall with and it's version.
+      placeholder: Pacstall 1.7.2 (develop)
```
Comment (D-Brox) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml]:
```suggestion
      placeholder: LibPacstall 0.1.0
```

**Review by D-Brox — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml (hunk 1 above)
Comment (wizard-28) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml]: People won't use the library with the library though. See the description.

Comment (D-Brox) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml]: Ok, true, but we need the version of libpacstall too

Comment (wizard-28) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml]: Right how about something like this:

---

# LibPacstall version

What LibPacstall version are you using?

You can find the LibPacstall version with:

```console
python3 -c "import libpacstall; print(libpacstall.__version__)"
```

*Input box here*

Comment (D-Brox) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml]: I'd use:
You can find the LibPacstall version with `python3 -c "import libpacstall; print(libpacstall.__version__)"`
In one single line.
But ye, seem good to me

Comment (wizard-28) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml]: @D-Brox my one would look like this:



If we put it in one line we would loose the syntax highlighting.

Comment (D-Brox) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml]: You can add it then

Comment (wizard-28) [reply .github/ISSUE_TEMPLATE/BUG-REPORT.yml]: Resolved in https://github.com/pacstall/libpacstall/pull/18/commits/e68192cbe191a6a20afb0f8c84348e16f53629f4

**Review by D-Brox — APPROVED** [review]

**Pull request merged**